packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -107,12 +116,23 @@ export default function vercelServerless({
 	return {
 		name: PACKAGE_NAME,
 		hooks: {
-			'astro:config:setup': ({ command, config, updateConfig, injectScript }) => {
-				if (command === 'build' && analytics) {
-					injectScript('page', 'import "@astrojs/vercel/analytics"');
+			'astro:config:setup': async ({ command, config, updateConfig, injectScript }) => {
+				if (webAnalytics?.enabled) {
+					injectScript(
+						'page',
```

Comment:
Does that mean that analytics are not used for endpoints and redirects? If this is intentional, I think we should document it.

Reply:
That's how analytics typically works. It requires a client side script, hence why we can't track these things.

Reply:
Thank you! :)